rel-eng/tito.props

```diff
@@ -403,18 +403,6 @@ whitelist = ansiblerole-insights-client
   rubygem-smart_proxy_remote_execution_ssh
   rubygem-smart_proxy_salt
 
-[katello-nightly-rhel5]
-disttag = .el5
-whitelist = katello-host-tools
-  katello-repos
-  python-rhsm
-  subscription-manager
-
-[katello-nightly-rhel6]
-disttag = .el6
-whitelist = katello-host-tools
-  katello-repos
-
 [katello-nightly-rhel7]
```

Comment:
I'm wondering if we should split this in `katello-server-nightly-rhel7` and `katello-client-nightly-rhel7` but that's probably out of scope for this PR.

Reply:
I've considered that as well, and out of scope.